scalability_and_performance/cnf-numa-aware-scheduling.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: ASSEMBLY
+[id="cnf-numa-aware-scheduling"]
+= NUMA-aware scheduling with NUMA Resources Operator
+include::modules/common-attributes.adoc[]
+:context: numa-aware
+
+toc::[]
+
+Learn about NUMA-aware scheduling and how you can use it to deploy high performance workloads in an {product-title} cluster.
+
+:FeatureName: NUMA-aware scheduling
+include::snippets/developer-preview.adoc[leveloffset=+1]
+
+[IMPORTANT]
+.NUMA-aware scheduling Developer Preview limitations and known issues
+====
+. The NUMA-aware scheduling stack does not support the `container` topology manager scope. For more information, see link:https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#topology-manager-scopes[Topology manager scopes].
+. Alignment of memory resources is not supported. For more information, see link:https://kubernetes.io/docs/tasks/administer-cluster/memory-manager/[Utilizing the NUMA-aware memory manager].
+====
+
+The NUMA Resources Operator allows you to schedule high-performance workloads in the same NUMA zone. It deploys a node resources exporting agent that reports on available cluster node NUMA resources, and a secondary scheduler that manages the workloads.
+
+include::modules/cnf-about-numa-aware-scheduling.adoc[leveloffset=+1]
+
+.Additional resources
+
+* For more information about running custom pod schedulers in your cluster and how to deploy pods with a custom pod scheduler, see xref:../nodes/scheduling/nodes-custom-scheduler.adoc#nodes-custom-scheduler[Running a custom scheduler].
+
+include::modules/cnf-installing-numa-resources-operator.adoc[leveloffset=+1]
+
```

Comment:
do we want to add the new file concerning creating nrop CR here? I'd say this is the most used flow:
install operator
create nrop CR
create scheduler CR 
then verifying all components are up and running as mention already in `cnf-deploying-the-numa-aware-scheduler.adoc` line [140](https://github.com/openshift/openshift-docs/pull/40307/files#diff-9d70a7cbb13f9f6f34cc5f34113f77c652ae3b30704419b14c41ac21a1a336cfR140)

Reply:
Doesn't this logically follow under the **Deploying the NUMA-aware secondary pod scheduler** section? ie we already have this material there, correct me if I am wrong. 
https://deploy-preview-40307--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-numa-aware-scheduling.html#cnf-deploying-the-numa-aware-scheduler_numa-aware